src/main/java/com/kobylynskyi/graphql/codegen/GraphQLCodegen.java

```diff
@@ -255,9 +255,13 @@ private void validateConfigs(MappingConfig mappingConfig) {
         MappingContext context = new MappingContext(mappingConfig, document, generatedInformation);
 
         List<File> generatedFiles = new ArrayList<>();
+        // Enumeration must be first for scala, interface must before type for kotlin/scala!!!
```

Comment:
Because Scala's enumerations need to be imported for use, they may not be generated before the type is generated

kotlin also, for override

Reply:
You need to know which interfaces need to be rewritten to add 'override'. Otherwise, it cannot be implemented.
But Scala and Java's `override` is optional, which is also due to kotlin's strict restrictions

Reply:
Use document to pre calculate and no longer rely on order.